src/test/java/com/googleinterns/zoomtube/servlets/AuthenticationServletTest.java

```diff
@@ -0,0 +1,129 @@
+package com.googleinterns.zoomtube.servlets;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
```

Comment:
These need to be alphabetized.

Reply:
Updated linter to fix these formatting issues